scripts/linters/python_linter.py

```diff
@@ -117,6 +117,44 @@ def _check_import_order(self):
                 summary_messages.append(summary_message)
         return summary_messages
 
+    def _check_non_test_files(self):
+        """This function is used to check that function
+           with test_only in their names are in test files.
+        """
+        if self.verbose_mode_enabled:
+            python_utils.PRINT('Starting function defintion checks')
+            python_utils.PRINT('----------------------------------------')
+        summary_messages = []
+        files_to_check = self.py_filepaths
+        with linter_utils.redirect_stdout(sys.stdout):
+            failed = False
+            for filepath in files_to_check:
+                if not filepath.endswith('_test.py'):
+                    file_content = FILE_CACHE.read(filepath)
+                    for line_number, line in enumerate(
+                            file_content.split('\n')):
+                        ind1 = line.find(' def ')
+                        ind2 = line.find('test_only')
+                        if ind1 != -1 and ind2 != -1:
+                            python_utils.PRINT(
```

Comment:
assign it to `summary_message` then print the `summary_message`, append to `summary_messages` as well.

Reply:
But summary messages are going to be printed on console after all checks right?
so in summary messages I only need to tell whether the checks are passed or not, I don't understand why are we appending the failing file info in the summary messages?Am i missing something?

Reply:
Summary messages format is going to change in a future PR. So, for that, we need this change.

Reply:
done